test/Makefile

```diff
@@ -3,6 +3,14 @@ ifdef OPENMP
   CFLAGS += -fopenmp
 endif
 
+TARGET := $(shell $(CC) -dumpmachine)
```

Comment:
Ah, I just remembered that the Windows builds are created using CMake, so my first point is moot.